config/application.yml.default

```diff
@@ -228,7 +228,7 @@ development:
   resolution_proof_result_lambda_token: ABC123
   s3_report_bucket_prefix:
   s3_reports_enabled: 'false'
-  saml_endpoint_configs: '[{"suffix":"2019","secret_key_passphrase":"trust-but-verify"},{"suffix":"2018","secret_key_passphrase":"asdf1234"},{"suffix":"2020","secret_key_passphrase":"trust-but-verify"}]'
+  saml_endpoint_configs: '[{"suffix":"2020","secret_key_passphrase":"trust-but-verify"},{"suffix":"2021","secret_key_passphrase":"trust-but-verify"}]'
```

Comment:
Speaking of tests, I'm pretty sure updating the test endpoint configs will resolve most of the test failures. Some of the SAML helper may need work as well